tfhe/src/integer/server_key/radix/bitwise_op.rs

```diff
@@ -207,23 +198,33 @@ impl ServerKey {
     where
         T: IntegerRadixCiphertext,
     {
-        if !self.is_functional_bivariate_pbs_possible(ct_left, ct_right) {
+        if self
+            .is_functional_bivariate_pbs_possible(ct_left, ct_right)
+            .is_err()
+        {
             self.full_propagate(ct_left);
             self.full_propagate(ct_right);
         }
-        assert!(self.is_functional_bivariate_pbs_possible(ct_left, ct_right));
+        assert!(self
+            .is_functional_bivariate_pbs_possible(ct_left, ct_right)
+            .is_ok());
```

Comment:
Shouldn't these be unwrap ?

Reply:
Yes, thanks!
I guess my regex did not allow newlines :)

Reply:
Done